src/tooltip/tooltip.directive.ts

```diff
@@ -175,8 +179,10 @@ export class TooltipDirective implements OnInit, OnDestroy {
   /** @deprecated */
   @Output()
   tooltipStateChanged: EventEmitter<boolean> = new EventEmitter<boolean>();
-  protected _delayTimeoutId: number | any;
-  protected _tooltipCancelShowFn: Function;
+  protected _obsHide = Observable.of<number>(-1);
```

Comment:
can we inline this values if they used only in one place?
or move them on top of file to not create for each tooltip instance?